lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -78,6 +78,27 @@ class EmptyXModuleDescriptor(XModuleDescriptor):  # pylint: disable=abstract-met
     module_class = EmptyXModule
 
 
+class GradedStatelessXBlock(XBlock):
+    """
+    This XBlock exists to test grade storage for blocks that don't store
+    student state in a scoped field.
+    """
+
+    @XBlock.json_handler
+    def set_score(self, json_data, suffix):  # pylint: disable=unused-argument
+        """
+        Set the score for this testing XBlock.
+        """
+        self.runtime.publish(
+            self,
+            'grade',
+            {
+                'value': json_data['grade'],
+                'max_value': 1
+            }
+        )
```

Comment:
So what's written to the state field in CSM when you set_score() on something with no student state?